libxss will fail pkgconf test pipeline

If the pkgconf test pipeline is added to libxss it will fail in the following way:

```
2024/11/12 23:12:59 INFO running step "test/pkgconf"
2024/11/12 23:12:59 WARN + '[' -d /home/build ] uses=test/pkgconf
2024/11/12 23:12:59 WARN + cd /home/build uses=test/pkgconf
2024/11/12 23:12:59 WARN + exit 0 uses=test/pkgconf
2024/11/12 23:12:59 INFO running step "pkgconf build dependency check" uses=test/pkgconf
2024/11/12 23:12:59 WARN + '[' -d /home/build ] uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:59 WARN + cd /home/build uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:59 WARN + basename /home/build/melange-out/libxss-dev uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:59 WARN + dev_pkg=libxss-dev uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:59 WARN + cd / uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:59 WARN + apk info -L libxss-dev uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:59 WARN + grep '\.pc$' uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:59 WARN WARNING: opening /home/user/src/wolfi-os/packages: No such file or directory uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:59 WARN WARNING: opening from cache https://packages.wolfi.dev/os: No such file or directory uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:59 WARN + grep -q ^Name: usr/lib/pkgconfig/xscrnsaver.pc uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:59 WARN + basename usr/lib/pkgconfig/xscrnsaver.pc .pc uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:59 WARN + lib_name=xscrnsaver uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:59 WARN + echo usr/lib/pkgconfig/xscrnsaver.pc uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:59 WARN + grep -q '^usr/lib/pkgconfig/xscrnsaver.pc$\|^usr/share/pkgconfig/xscrnsaver.pc$' uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:59 WARN + pkgconf --exists xscrnsaver uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:59 WARN + pkgconf --print-errors --exists xscrnsaver uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:59 WARN Package scrnsaverproto was not found in the pkg-config search path. uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:59 WARN Perhaps you should add the directory containing `scrnsaverproto.pc' uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:59 WARN to the PKG_CONFIG_PATH environment variable uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:59 WARN Package 'scrnsaverproto', required by 'xscrnsaver', not found uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:59 INFO pod d5dfa40543486d2d5b9a09567932d29bda989e45deac0ddeb00ee2124a12dd92 terminated
2024/11/12 23:12:59 INFO ERROR: failed to test package. the test environment has been preserved:
2024/11/12 23:12:59 INFO   workspace dir: /home/user/tmp/melange-workspace-1662990137
2024/11/12 23:12:59 INFO   guest dir: /home/user/tmp/melange-guest-4249455108
2024/11/12 23:12:59 ERRO failed to test package: unable to run pipeline: unable to run pipeline: unable to run pipeline: task exited with code 1
make: *** [Makefile:147: test/libxss] Error 1
```

--- a/libxss.yaml
+++ b/libxss.yaml
@@ -1,7 +1,7 @@
 package:
   name: libxss
   version: 1.2.4
-  epoch: 1
+  epoch: 2
   description: "Xlib-based X11 Screen Saver extension client library"
   copyright:
     - license: XFree86-1.1
@@ -31,8 +31,14 @@ pipeline:
 
 subpackages:
   - name: libxss-dev
+    dependencies:
+      runtime:
+        - libx11-dev
     pipeline:
       - uses: split/dev
+    test:
+      pipeline:
+        - uses: test/pkgconf
 
   - name: libxss-doc
     pipeline:

